patches/net/minecraft/world/item/ProjectileWeaponItem.java.patch

```diff
@@ -1,5 +1,34 @@
 --- a/net/minecraft/world/item/ProjectileWeaponItem.java
 +++ b/net/minecraft/world/item/ProjectileWeaponItem.java
+@@ -24,12 +_,28 @@
+         super(p_43009_);
+     }
+ 
++    /**
++     * Use ItemStack sensitive version instead
++     */
++    @Deprecated
+     public Predicate<ItemStack> getSupportedHeldProjectiles() {
+         return this.getAllSupportedProjectiles();
+     }
+ 
++    /**
++     * Use ItemStack sensitive version instead
++     */
++    @Deprecated
+     public abstract Predicate<ItemStack> getAllSupportedProjectiles();
+ 
++    public Predicate<ItemStack> getSupportedHeldProjectiles(ItemStack stack) {
```

Comment:
Please add some javadoc describing what this should do and what the stack parameter is.

Reply:
resolved in commit #a091806